From 67d28ac629e96912bb7152bbb172217d0f1c70c0 Mon Sep 17 00:00:00 2001
From: "Richard W.M. Jones" <rjones@redhat.com>
Date: Thu, 23 Oct 2014 08:41:56 +0100
Subject: [PATCH] RHEL 7: v2v: Disable unconfiguration of VMware drivers on
 Linux (RHBZ#1155610).

This is currently broken and breaks conversions.  For details see:
https://bugzilla.redhat.com/show_bug.cgi?id=1155610
---
 v2v/convert_linux.ml | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/v2v/convert_linux.ml b/v2v/convert_linux.ml
index 709b4b9..67de2b1 100644
--- a/v2v/convert_linux.ml
+++ b/v2v/convert_linux.ml
@@ -537,6 +537,7 @@ let rec convert ~verbose ~keep_serial_console (g : G.guestfs) inspect source =
               msg
     )
 
+(* Disabled in RHEL 7.1: see https://bugzilla.redhat.com/show_bug.cgi?id=1155610
   and unconfigure_vmware () =
     (* Look for any configured VMware yum repos and disable them. *)
     let repos =
@@ -627,6 +628,7 @@ let rec convert ~verbose ~keep_serial_console (g : G.guestfs) inspect source =
           warning ~prog (f_"VMware tools was detected, but uninstallation failed.  The error message was: %s (ignored)")
             msg
     )
+*)
 
   and unconfigure_citrix () =
     let pkgs =
@@ -1417,7 +1419,7 @@ let rec convert ~verbose ~keep_serial_console (g : G.guestfs) inspect source =
 
   unconfigure_xen ();
   unconfigure_vbox ();
-  unconfigure_vmware ();
+  (*unconfigure_vmware ();*)
   unconfigure_citrix ();
   unconfigure_efi ();
   unconfigure_kudzu ();
-- 
1.8.3.1

